lms/djangoapps/verify_student/signals.py

```diff
@@ -18,7 +18,7 @@
 
 
 # Signal for emitting IDV submission and review updates
-idv_update_signal = Signal(providing_args=["attempt_id", "user_id", "status", "full_name", "profile_name"])
+idv_update_signal = Signal()
```

Comment:
providing arguments should be added in comments.

Reply:
Looks like this is still needed, the rest of the PR looks fine.

Reply:
Updated.